### DISCUSSION TITLE
Time not properly detected by Xarray in ISSM (Ice Sheet System Model) simulations

### DISCUSSION THREAD

**Discussion by jbbarre** [jbbarre/ISM_SimulationChecker#13] [Q&A]
hello @hseroussi,

Regarding the error of time related verifications in ISSM (Ice Sheet System Model) simulations, It comes from the declaration of the unit in the netcdf file that does not comply  with the CF conventions.  IT shoulf be` time:units `instead of `time:unit`. That little 's' makes all the difference. We have tested your data sample with this modification and the checker works successfully without any errors.

**Comment by hseroussi** [comment]
Thank you @jbbarre 

I just changed that and it indeed works perfectly now. I had checked the name of the units many times, but not the word units itself. I will keep that in mind if other people have problems.
